tensilelite/Tensile/Components/GlobalWriteBatch.py

```diff
@@ -1993,9 +1993,15 @@ def _addSumAlphaWithCBeta(self, kernel, ss, gwvw, elementIdx, vc0, tmpVgpr, cvtV
           # Generate single f32 code if edge is detected.
           isPK = False
           if ((vi + 1) == self.gwvw) and ((self.gwvw % 2) == 1):
-            if self.parentWriter.states.archCaps["NoSDWA"]:
+            if self.parentWriter.states.archCaps["VOP3ByteSel"]:
               sb = 0 if self.gwvw == 1 else 1
-              module.add(VCvtFP8toF32(dst=vgpr(tmpVgpr), src=vgpr(dataV), vop3=VOP3PModifiers(op_sel=[0,sb])))
+              hipccver = globalParameters['HipClangVersion'].split(".")
+              hipccMaj = int(hipccver[0])
+              hipccMin = int(hipccver[1])
+              if not (hipccMaj >= 6 and hipccMin >= 4):
```

Comment:
Are you sure that all hipccMin >= 4 version contains the compiler changes?
Do we have exact hipccPatch that compiler starts to support it?

Reply:
I only saw Mark suggest us checking Clang version.
Have you verified it in internal docker by only check hipcc >= 6.4?